prow/config/config.go

```diff
@@ -249,7 +255,61 @@ type Branding struct {
 }
 
 // Load loads and parses the config at path.
-func Load(path string) (*Config, error) {
+func Load(prowConfig, jobConfig string) (*Config, error) {
+	stat, err := os.Stat(prowConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	nc := &Config{}
+	if !stat.IsDir() {
+		nc, err = yamlToConfig(prowConfig)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	// TODO(krzyzacy): temporary allow empty jobconfig
+	//                 also temporary allow job config in prow config
+	if jobConfig == "" {
+		return nc, nil
+	}
+
+	stat, err = os.Stat(jobConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	if !stat.IsDir() {
+		// jobConfig need to be in a directory
+		return nil, err
+	}
+
+	err = filepath.Walk(jobConfig, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			fmt.Printf("prevent panic by handling failure accessing a path %q: %v\n", path, err)
+			return err
+		}
+
```

Comment:
Skip directories.
```go
if info.IsDir() {
  return nil
}
```
Should we support multiple directory levels instead of just a flat dir?

Reply:
yes we should

Reply:
it should be covered with `if filepath.Ext(path) != "yaml"` check

Reply:
That seems reasonable. If someone makes a directory with a `.yaml` suffix they're asking for trouble anyways.

Reply:
done